homework5/task1.js

```diff
@@ -0,0 +1,86 @@
+function PersonFunc(name, lastname, birth, address) {
+
+    this.firstName = name,
+        this.lastName = lastname,
+        this.age = birth,
+        this.address = address;
+
+    this.fullName = function () {
+        try {
+            if (!this.firstName || !this.lastName) {
+                throw new SyntaxError("First name or Full name is missing");
+            }
+            console.log(this.firstName + " " + this.lastName)
+
+        } catch (e) {
+
+            if (e.name == "SyntaxError") {
+                console.log("First name or Full name is missing");
+            } else {
+                throw e;
+            }
+
+        }
+    }
+    this.getAge = function () {
+        try {
+            if (!this.age) {
+                throw new SyntaxError("Birth date is missing");
+            }
+            let age = this.age
+            let arr = age.split('.')
+            let result = 2022 - arr[arr.length - 1]
+            console.log(result + " " + "years")
+
+        } catch (e) {
+
+            if (e.name == "SyntaxError") {
+                console.log("Birth date is missing");
```

Comment:
and here
Don't need to leave consoles for results inside the function
Better is to use console.log outside